webpack.config.js

```diff
@@ -26,21 +30,30 @@ module.exports = function (env) {
     }),
   ];
   return {
-    mode: isProd ? "production" : "development",
     entry: {
-      main: "/static/js/src/RecentListens.tsx",
-      import: "/static/js/src/LastFMImporter.tsx",
-      userEntityChart: "/static/js/src/stats/UserEntityChart.tsx",
-      userReports: "/static/js/src/stats/UserReports.tsx",
-      userPageHeading: "/static/js/src/UserPageHeading.tsx",
-      userFeed: "/static/js/src/user-feed/UserFeed.tsx",
-      playlist: "/static/js/src/playlists/Playlist.tsx",
-      playlists: "/static/js/src/playlists/Playlists.tsx",
-      recommendations: "/static/js/src/recommendations/Recommendations.tsx",
+      // Importing main.less file here so that it gets compiled.
+      // All the Less/CSS will be exported separately to a main.css file
```

Comment:
maybe one more comment saying that we don't have a standalone entrypoint for the less file so that it doesn't generate a superfluous js file